.github/policies/labelAdded.mapQuest.yml

```diff
@@ -0,0 +1,32 @@
+id: labelAdded.mapQuest
+name: GitOps.MapQuestHandler
+description: Handles when "mapQUEST" label is added
+owner:
+resource: repository
+disabled: false
+where:
+configuration:
+  resourceManagementConfiguration:
+    eventResponderTasks:
+      - description: >-
+          When the label "mapQUEST" is added to an issue
+          * Remove label "mapQUEST"
+        if:
+          - payloadType: Issues
```

Comment:
I think you can add more than one payload type and then just combine these into a single task.

Reply:
Here's an example: https://github.com/microsoft/testfx/blob/abc7029b7a948ca213f176713c09ace9553e7880/.github/policies/LabelManagement.IssueUpdated.yml#L14-L19